check_run/public/js/vite.config.js

```diff
@@ -6,21 +6,20 @@ import path from 'path'
 export default defineConfig({
 	plugins: [vue()],
 	build: {
-		lib: {
-			entry: path.resolve(__dirname, './check_run/check_run.js'),
-			name: 'check_run',
-			fileName: () => `check_run.js`, // creates module only output
-		},
 		outDir: './check_run/public/dist/js',
-		root: './',
-		target: 'es2015',
 		emptyOutDir: false,
-		minify: false,
+		manifest: true,
 		rollupOptions: {
+			input: {
+				check_run: path.resolve(__dirname, './check_run/check_run.js'),
+			},
 			output: {
-				chunkFileNames: 'chunks/[name].[hash].js',
+				entryFileNames: 'check_run_vue.bundle.[hash].js',
+				assetFileNames: 'style.css',
+				format: 'iife',
```

Comment:
The `style.css` file is no longer generated by Vite, and is throwing errors on the UI. Maybe removing the `format` key entirely, or changing it to `esm` would solve that?